.github/workflows/breaking-change-review.yml

```diff
@@ -0,0 +1,145 @@
+name: PR Review Workflow for Breaking Changes
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - labeled
+      - unlabeled
+      - synchronize
+
+jobs:
+  check_breaking_change:
```

Comment:
A human-readable name would be nice as well:
```suggestion
  check_breaking_change:
    name: Require QA review
```

This is what we have for conventional commits:

<img width="558" alt="image" src="https://github.com/user-attachments/assets/94c20f66-e590-4204-aca1-a96f920d8b20">